config/analytics_hidden_pii.yml

```diff
@@ -0,0 +1,15 @@
+---
+shared:
+  application_forms:
+    - support_reference
+    - candidate_id
+  candidates:
+    - id
```

Comment:
Data insights team is looking at the application.candidate_id that is a foreign key of the candidates.id, if we anonymise both of these then they might loose the link between the two, they are checking. 

Please do not merge until this is confirmed.

Reply:
Data Insights confirmed that there will be no issue with the application.candidate_id and candidates.id being anonymised